### PR TITLE
Update to Rubocop 0.80

### PR DESCRIPTION
**Pull request by josemiguelmelo**
This will add support for Rubocop 0.80. 
Some cops were rename on this new version.

### CODE REVIEW

File: rubocop-airbnb/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.3
```
Comment (ljharb) [inline rubocop-airbnb/CHANGELOG.md]: version numbers are only decided at publish time. (also, adding support for a rubocop version is at least semver-minor, not patch)
```suggestion
# Unreleased
```

File: rubocop-airbnb/lib/rubocop/airbnb/version.rb [hunk 2]
```diff
@@ -3,6 +3,6 @@
 module RuboCop
   module Airbnb
     # Version information for the the Airbnb RuboCop plugin.
-    VERSION = '3.0.2'
+    VERSION = '3.0.3'
```
Comment (ljharb) [inline rubocop-airbnb/lib/rubocop/airbnb/version.rb]:
```suggestion
    VERSION = '3.0.2'
```

Comment (josemiguelmelo) [reply rubocop-airbnb/lib/rubocop/airbnb/version.rb]: 👍

File: rubocop-airbnb/CHANGELOG.md (hunk 1 above)
Comment (josemiguelmelo) [reply rubocop-airbnb/CHANGELOG.md]: You're right, done 👍

**Comment by ljharb** [comment]
This LGTM, but it seems like renaming the rules makes this semver-major?

**Comment by josemiguelmelo** [comment]
Yes, you're absolutely right! 👍

**Comment by ljharb** [comment]
nope, just whoever merges and releases it needs to know that :-)

**Comment by lolgab** [comment]
Any plans to merge this? Thank you.

**Comment by ljharb** [comment]
It would require someone with write access on the repo to approve, and click the button. That hasn't been me in the better part of a year ¯\\\_(ツ)_/¯

**Comment by josemiguelmelo** [comment]
Are any plans to merge this? 
Thanks 😊

**Comment by zachsabin** [comment]
@josemiguelmelo sorry for the delay, I will look at this now

**Comment by josemiguelmelo** [comment]
Thank you for merging this pull request 🙏

**Pull request merged**